examples/swap/swap-e2e.html

```diff
@@ -95,12 +96,12 @@ <h3>Claim swap:</h3>
     /* global $, ChainAbstractionLayer, web3 */
     const { Client, providers, networks, crypto } = ChainAbstractionLayer
     const bitcoin = new Client()
-    bitcoin.addProvider(new providers.bitcoin.BitcoinRPCProvider('https://localhost:18332', 'bitcoin', 'local321'))
-    bitcoin.addProvider(new providers.bitcoin.BitcoinLedgerProvider({ network: networks.bitcoin_testnet }))
+    bitcoin.addProvider(new providers.bitcoin.BitcoinRPCProvider('http://localhost:8000', 'bitcoin', 'local321'))
```

Comment:
you don't have issues running without https? 🤔 
Can't remember for what reason but I had to make it run over https, might have been because of ledger, having to run the example through https

i'd keep the default bitcoin port in the example that's committed to master

Reply:
Maybe we should actually have a config file or ENV vars for this

Reply:
Reverted for now